source-mysql-batch/driver.go

```diff
@@ -595,11 +574,39 @@ func (c *capture) poll(ctx context.Context, bindingIndex int, tmpl *template.Tem
 	}
 	var query, args = expandQueryPlaceholders(queryBuf.String(), cursorValues)
 
+	// Polling interval can be configured per binding. If unset, falls back to the
+	// connector global polling interval.
+	var pollStr = c.Config.Advanced.PollInterval
+	if res.PollInterval != "" {
+		pollStr = res.PollInterval
+	}
+	pollInterval, err := time.ParseDuration(pollStr)
+	if err != nil {
+		return fmt.Errorf("invalid poll interval %q: %w", res.PollInterval, err)
+	}
+
+	// Sleep until it's been more than <pollInterval> since the last iteration,
+	// then update the "Last Polled" timestamp.
+	if !state.LastPolled.IsZero() && time.Since(state.LastPolled) < pollInterval {
+		select {
```

Comment:
nit: add a log of when we will next awake to poll the table?

Reply:
Done